### PR TITLE
Add jontonsoup userspace keymap 

### PR DESCRIPTION
**Pull request by jontonsoup**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

**Comment by drashna** [comment]
It looks like you need to rebase your branch:
https://docs.qmk.fm/#/newbs_git_resolving_merge_conflicts

**Comment by jontonsoup** [comment]
Whoops! I think this was because I rebased and changed the order of my commits improperly. Will fix. Thanks :)

**Comment by drashna** [comment]
Not a problem. Git is ... weird sometimes. But looks like you got it!

File: keyboards/keebio/iris/keymaps/jontonsoup/keymap.c [hunk 1]
```diff
@@ -0,0 +1,130 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/jontonsoup/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/keebio/iris/keymaps/jontonsoup/keymap.c [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/jontonsoup/keymap.c:7]: He highly recommend using an enum, here. 

```suggestion
enum layers {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keebio/iris/keymaps/jontonsoup/keymap.c (hunk 1 above)
Comment (jontonsoup) [reply keyboards/keebio/iris/keymaps/jontonsoup/keymap.c:1]: Apologies -- I just forked the main repo and followed the directions (maybe I missed this part). Is the idea here that each contributor needs to add a license vs the entire program is not covered by a license? 

Thanks for taking the time to review this!

Comment (drashna) [reply keyboards/keebio/iris/keymaps/jontonsoup/keymap.c:1]: Basically, yup.  Specifically: 
https://www.gnu.org/licenses/gpl-faq.en.html#LicenseCopyOnly
https://www.gnu.org/licenses/gpl-faq.en.html#NoticeInSourceFile

The tl;dr: is that even if the license file exists in the repo, that doesn't cover the individual files if somebody copies them elsewhere.   

So to be explicitly clear on the licensing, it's best to include it in the header for each file.

**Pull request closed without merging**